package.json

```diff
@@ -3,9 +3,9 @@
   "private": true,
   "version": "3.5.7",
   "description": "🐻 Bear necessities for state management in React",
-  "main": "index.js",
-  "module": "esm/index.js",
-  "types": "index.d.ts",
+  "main": "./index.js",
+  "module": "./esm/index.js",
+  "types": "./index.d.ts",
```

Comment:
Let's make this change in https://github.com/pmndrs/jotai/pull/657 too.

Reply:
Done